crates/zed/src/app_menus.rs

```diff
@@ -40,7 +40,7 @@ pub fn app_menus() -> Vec<Menu<'static>> {
                 MenuItem::action(
                     "Open Recent...",
                     recent_projects::OpenRecent {
-                        create_new_window: false,
+                        create_new_window: true,
```

Comment:
`false` was used as a default due to the same https://github.com/zed-industries/zed/issues/8651#issuecomment-1973390500